tests/sentry/issues/endpoints/test_organization_group_search_views.py

```diff
@@ -76,11 +76,11 @@ def create_view(
                 view=view,
             )
         if last_visited:
-            GroupSearchViewLastVisited.objects.create(
-                user_id=user.id,
+            GroupSearchViewLastVisited.objects.create_or_update(
+                user_id=self.user.id,
```

Comment:
The test util was attributing `last_visited` to the incoming user instead of the current user, which was what hid this error and caused the tests to pass.